library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -20,42 +19,35 @@
 
 public final class SimpleChromeCustomTabs implements WebNavigator, Connection, AvailableAppProvider {
 
-    private static Context applicationContext;
     private Connection connection;
     private WebNavigator webNavigator;
     private AvailableAppProvider availableAppProvider;
 
-    private SimpleChromeCustomTabs() {
-        //no-op
-    }
-
-    private static class LazyHolder {
-        private static final SimpleChromeCustomTabs INSTANCE = new SimpleChromeCustomTabs();
+    /**
+     * @deprecated There is no need to call it anymore in order to use this library,
```

Comment:
yes there is no need, but its not deprecated, personally we should still be asking people to call it if we want to keep public api consistency